### PR TITLE
Add same version flag to docker-version-compare.

### PR DESCRIPTION
**Pull request by jefchien**
**Description:** This is to account for reruns of the CD.

### CODE REVIEW

**Review by sethAmazon — APPROVED** [review]

File: .github/workflows/CD.yml [hunk 1]
```diff
@@ -594,7 +594,7 @@ jobs:
           dst: ${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}:${{ needs.release-checking.outputs.version }}
 
       - name: Release adot operator
-        if: ${{ steps.release-latest-image.outputs.cache-hit != 'true' && (steps.version.outputs.major-update == 'true' || steps.version.outputs.minor-update == 'true') }}
+        if: ${{ steps.release-latest-image.outputs.cache-hit != 'true' && (steps.version.outputs.major-update == 'true' || steps.version.outputs.minor-update == 'true') || steps.version.outputs.same-version == 'true' }}
```
Comment (khanhntd) [inline .github/workflows/CD.yml:597]: I don't think you outputs the ```same version``` in this commit though.

Comment (jefchien) [reply .github/workflows/CD.yml:597]: Good catch. I lost it in the merge.

File: Makefile [hunk 2]
```diff
@@ -151,7 +151,7 @@ fmt:
 
 .PHONY: fmt-sh
 fmt-sh: $(SHFMT)
-	shfmt -w -d -i 4 .
+	$(SHFMT) -w -d -i 4 .
```
Comment (khanhntd) [inline Makefile:154]: Nice catch on fixing me error though.

**Review by khanhntd — APPROVED** [review]
LGTM.

**Pull request merged**